server/server_test.go

```diff
@@ -355,6 +355,50 @@ func TestServiceRegistryNoInitInfo(t *testing.T) {
 	test.Assert(t, drCount == 1)
 }
 
+func TestServiceRegistryInfoWithOutTags(t *testing.T) {
```

Comment:
typo, Without not WithOut

Reply:
rename to `TestServiceRegistryInfoWithNilTags`, and add comment for this test case
// TestServiceRegistryInfoWithNilTags is to check the Tags val. If Tags of RegistryInfo is nil, the Tags of ServerBasicInfo will be assigned to Tags of  RegistryInfo